bims/templates/main_base.html

```diff
@@ -39,10 +42,12 @@
 {% block body_content %} {% endblock %}
 
 <!-- Bootstrap core JavaScript -->
+<!-- IntroJS -->
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/intro.js/2.9.3/intro.min.js"></script>
+    <script type="text/javascript" src="../static/js/views/introjs-tutorial.js"></script>
```

Comment:
use `{% static ... to load the static file

Reply:
Fixed.